sdk/core/azure-core/azure/core/_pipeline_client.py

```diff
@@ -123,24 +123,22 @@ def _build_pipeline(self, config, **kwargs): # pylint: disable=no-self-use
                 ContentDecodePolicy(**kwargs)
             ]
             if isinstance(per_call_policies, Iterable):
-                for policy in per_call_policies:
-                    policies.append(policy)
+                policies.extend(per_call_policies)
```

Comment:
(For your another PR), do we need list(per_call_policies) here?

Reply:
Not really. [`list.extend(iterable)`]( https://docs.python.org/3/tutorial/datastructures.html) by definition takes any iterable.